### PR TITLE
Add a <noscript> tag to inform user if scripts are disabled/blocked

### PR DESCRIPTION
**Pull request by KWierso**

### CODE REVIEW

File: tests/test_vcs_hg/code/Colon: name [hunk 1]
```diff
@@ -1 +0,0 @@
-Filename has a colon
```
Comment (erikrose) [inline tests/test_vcs_hg/code/Colon: name]: Did you mean to commit this?

Comment (KWierso) [inline tests/test_vcs_hg/code/Colon: name]: I did not mean to commit that. I don't even know how that got there. I cloned the repo, switched to a new bookmark, edited the other file, git add'd that file, committed and pushed.

Comment (KWierso) [inline tests/test_vcs_hg/code/Colon: name]: I don't even see that file locally...

Comment (KWierso) [inline tests/test_vcs_hg/code/Colon: name]: Must be a Windows thing because that file's named like that... I'll try repushing this from linux.

**Comment by KWierso** [comment]
There we go. Sorry about that, @erikrose

**Comment by erikrose** [comment]
Hmm, I can't actually get the noscript message to show up (in Safari, with JS off).

**Comment by erikrose** [comment]
Aha, it's falling behind the header box.

**Comment by erikrose** [comment]
There, I fixed it. I'll merge a separate PR and credit you.

**Comment by erikrose** [comment]
Fixed in 403a4ed. Thanks for the nudge, @KWierso!

**Comment by erikrose** [comment]
Should be deployed by the magic monkeys in a few tens of minutes.

**Pull request closed without merging**